.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
```

Comment:
If we use a preview version for the CI builds and an official release version for development, we may encounter issues with incompatibilities. Is there no official version supported here?